fabtests/prov/efa/src/rdm_rnr_read_cq_error.c

```diff
@@ -162,13 +163,16 @@ int main(int argc, char **argv)
 	if (!hints)
 		return EXIT_FAILURE;
 
-	while ((op = getopt(argc, argv, ADDR_OPTS INFO_OPTS CS_OPTS)) != -1) {
+	while ((op = getopt(argc, argv, "k" ADDR_OPTS INFO_OPTS CS_OPTS)) != -1) {
 		switch (op) {
 		default:
 			ft_parse_addr_opts(op, optarg, &opts);
 			ft_parseinfo(op, optarg, hints, &opts);
 			ft_parsecsopts(op, optarg, &opts);
 			break;
+		case 'k':
+			__use_rnr_hook = 1;
```

Comment:
I think it is probably better to reuse the `-I` option, which stands for number of iterations (or number of messages to send).

set the default to `(fi->rx_attr->size + expected_rnr_error)` at the beginning, so it can be overwritten by command line options.